### PR TITLE
FIrst try to add issue template

### PR DESCRIPTION
**Pull request by ultratoto14**
As I do not have the rights to use the repository workflow, trying in that way.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+name: Bug report
+about: Create a report to help us improve localtuya
+title: ''
+labels: ''
```
Comment (postlund) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: I suggest that you add bug here.

**Review by postlund — CHANGES_REQUESTED** [review]

**Comment by postlund** [comment]
@rospogrigio I suggest that you make me and/or @ultratoto14 admins so we can steer the ship until you have time. That way you can just forget about localtuya until a better point in time.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (ultratoto14) [reply .github/ISSUE_TEMPLATE/bug_report.md:5]: I was thinking about it, but as for the moment, we do not have a feature_request template, or a new_device template, should we do it know or automatically categorize once we have different templates ?

**Comment by rospogrigio** [comment]
> @rospogrigio I suggest that you make me and/or @ultratoto14 admins so we can steer the ship until you have time. That way you can just forget about localtuya until a better point in time.

Yeah, totally agree: as I said from the very beginning it was just a matter of time until you'd better take over. 
Doing it now, I am quite overwhelmed these days...

**Comment by ultratoto14** [comment]
@rospogrigio maybe in Settings -> Manage Access
image

**Comment by rospogrigio** [comment]
You are already listes as Collaborators, and there is no "Admin" options:
image

**Comment by postlund** [comment]
My bad, I was under the impression that the access levels were more fine grained that they are. Only the repository owner has super user power and there can only be one owner. Collaborators can't get the kind of permissions needed I'm afraid.

**Comment by ultratoto14** [comment]
@rospogrigio if you want @postlund to take the lead on this repository, it seems the only way is to transfer the [ownership](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/transferring-a-repository#:~:text=On%20GitHub%2C%20navigate%20to%20the,ownership%20of%20the%20repository%20to.) 

Seems that fine grained rights tuning is only for github enterprise.

**Comment by rospogrigio** [comment]
OK then, just one question: what will happen with the HACS integration? It is enlisted as rospogrigio/localtuya there, as far as I remember...

**Comment by ultratoto14** [comment]
@rospogrigio i created an empty repository and transfer you the ownership, will see, maybe you need to accept.

**Comment by rospogrigio** [comment]
@ultratoto14  Can't see it...

**Comment by ultratoto14** [comment]
@rospogrigio the repository https://github.com/ultratoto14/TestingRepository
In the settings, i currently have 
image

**Comment by ultratoto14** [comment]
Yes, so it renamed the repository ... f.ck

**Comment by rospogrigio** [comment]
So, I suggest we investigate if there will be any issue with HACS, or if renaming the ownership can be transferred easily...

**Comment by ultratoto14** [comment]
By the way, it seems that the PR is working, tested it on my own repo.

**Comment by postlund** [comment]
I guess we can transfer ownership and you immediately make a new fork of it @rospogrigio. That way HACS can point to the fork until we have updated the repo in HACS.

**Comment by rospogrigio** [comment]
It tells me that "**_postlund already has a repository in the mileperhour/localtuya-homeassistant network_**".

**Comment by ultratoto14** [comment]
If the repository is changed then all the users will also have to update their links.

**Comment by postlund** [comment]
I have removed my local fork now, so hopefully it should work. Tried to read up a bit and I think all previous references will still work. So if anyone tries to use the old addresses, they will automatically point towards my repo, which means we are fine. People should still update to the new address though, we included when it comes to HACS.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (postlund) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Good point. Maybe we should just grab the bull by the horns and fix all of them? I have a few in my pyatv repo that has worked pretty well (for inspiration):

https://github.com/postlund/pyatv/tree/master/.github/ISSUE_TEMPLATE

**Comment by rospogrigio** [comment]
@postlund repository transfer requested :wink ... oops I transferred it to the trout!!! 🤣

**Comment by postlund** [comment]
@rospogrigio I am suppose to confirm, but maybe we should take another route instead.  There's an organization for custom components here:

https://github.com/custom-components

I'm sort of taking ownership of the `remote_homeassistant` component as well and we will move it to that organization. Maybe that's a better way to deal with this as well?

**Review by postlund — APPROVED** [review]

**Comment by postlund** [comment]
Let's merge this and see what happens...

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (CloCkWeRX) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I had a go at a PR template in https://github.com/rospogrigio/localtuya/pull/782

**Pull request merged**